srcpkgs/medusa/template

```diff
@@ -0,0 +1,59 @@
+# Template file for 'medusa'
+pkgname=medusa
+version=0.4.3
+revision=1
+build_style=cmake
+configure_args="
+	-DMEDUSA_BUILD_WITH_OGDF=ON 
+	-DOGDF_PATH=/usr/include/ogdf 
+	-DCMAKE_BUILD_TYPE=Release
+	-DBOOST_ROOT=/usr/lib
```

Comment:
this will fail super nicely on cross-compilation.

use ${XBPS_CROSS_BASE}/usr

same for LIBRARYDIR, except it is ${XBPS_CROSS_BASE}/usr/lib

and INCLUDEDIR which is ${XBPS_CROSS_BASE}/usr/includeboost